### PR TITLE
PORTALS-2147: add /logout route

### PR DESCRIPTION
**Pull request by jay-hodgson**

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -53,11 +58,11 @@ const App: React.FC = () => {
...
                         } else if (path === '/authenticated/myaccount') {
-                        return <AccountSettings/>
+                          return <AccountSettings/>
                         } else {
-                        return (<>
-                          <p>Unrecognized match path {routeProps.match.path}</p>
-                        </>)
+                          return (<>
+                            <p>Unrecognized match path {routeProps.match.path}</p>
+                          </>)
```
Comment (jay-hodgson) [inline src/App.tsx:59]: Merge from upstream, had a few whitespace changes.

File: src/components/RegisterAccount1.tsx [hunk 2]
```diff
@@ -96,12 +97,12 @@ export const RegisterAccount1 = (props: RegisterAccount1Props) => {
...
-              alt="Sage Bionetworks Logo"
-              style={{width: 270}}
-            />
-            <div className="registrationPanel">
+                className="sageLogo"
+                src={SageBionetworksLogo}
+                alt="Sage Bionetworks Logo"
+                style={{width: 270}}
+              />
+            <div className="mainContent">
```
Comment (jay-hodgson) [inline src/components/RegisterAccount1.tsx:105]: renamed registrationPanel to mainContent and moved duplicate styles to _SourceApp.scss

File: src/App.tsx [hunk 3]
```diff
@@ -28,15 +30,12 @@ const App: React.FC = () => {
...
                   render={props => {
-                    return <>
-                      <p>There are a few main entrypoints into this web app</p>
-                      <p>
-                        <a href='/register1'>Account Registration</a>,&nbsp;
-                        <a href='/authenticated/validate'>Profile Validation</a>,&nbsp;and&nbsp;
-                        <a href='/authenticated/myaccount'>My Account</a>
-                      </p>
-                      </>
+                    return <Redirect to='/authenticated/myaccount' />
```
Comment (jay-hodgson) [inline src/App.tsx:33]: Got rid of the unstyled landing page, and redirect to My Account.  (Note that I updated the readme to show the available entry points into this application).

File: src/AppInitializer.tsx [hunk 4]
```diff
@@ -110,6 +114,10 @@ class AppInitializer extends React.Component<Props, AppInitializerState> {
           {React.Children.map(this.props.children, (child: any) => {
             return child
           })}
+          {/* Force synapse.org login state to be in sync with this app. 
+          Note that this is only rendered after hasCalledGetSession is set to true. */}
+          {this.state.token && <HiddenIFrame url={`https://signin.synapse.org/login?code=${this.state.token}`} />}
```
Comment (yjyoo773) [inline src/AppInitializer.tsx:119]: Could you explain what this is doing? From my understanding, when the user has a token it renders the HiddenIFrame which calls the signin for Synapse then signing in the user. Please correct me if I am wrong.

**Review by yjyoo773 — APPROVED** [review]

File: src/AppInitializer.tsx (hunk 4 above)
Comment (jay-hodgson) [reply src/AppInitializer.tsx:119]: That's correct, this will establish the session for all .synapse.org sites.

**Pull request merged**